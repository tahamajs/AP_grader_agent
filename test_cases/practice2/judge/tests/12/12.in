reserve_table 1 6
reserve_table 2 6
reserve_table 3 6
reserve_table 4 6
reserve_table 5 6
reserve_table 6 6
reserve_table 7 6
reserve_table 8 6
reserve_table 9 6
reserve_table 10 6
exit 1
exit 2
show_table_info 6
exit 3
exit 4
show_table_info 6
reserve_table 2 6
exit 8
show_table_info 6
reserve_table 1 6
exit 9
show_table_info 6
exit 2
exit 7
exit 10
show_table_info 6
