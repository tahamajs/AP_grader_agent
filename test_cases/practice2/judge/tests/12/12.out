Paria sits at table 6
Amir Hossein sits at table 6
Parsa sits at table 6
Mehrad enters the waiting queue of table 6
Fereshte enters the waiting queue of table 6
Amir Akbar enters the waiting queue of table 6
Mahdi enters the waiting queue of table 6
Taha enters the waiting queue of table 6
Sadegh enters the waiting queue of table 6
Hamid enters the waiting queue of table 6
Paria exits!
Amir Hossein exits!
Table ID: 6
People at the table: Mahdi, Mehrad, Parsa
Table remaining capacity: 0
Waiting queue length: 5
Parsa exits!
Mehrad exits!
Table ID: 6
People at the table: Mahdi, Sadegh, Taha
Table remaining capacity: 0
Waiting queue length: 3
Amir Hossein enters the waiting queue of table 6
Taha exits!
Table ID: 6
People at the table: Amir Hossein, Mahdi, Sadegh
Table remaining capacity: 0
Waiting queue length: 3
Paria enters the waiting queue of table 6
Sadegh exits!
Table ID: 6
People at the table: Amir Hossein, Hamid, Mahdi
Table remaining capacity: 0
Waiting queue length: 3
Amir Hossein exits!
Mahdi exits!
Hamid exits!
Table ID: 6
People at the table: Amir Akbar, Fereshte, Paria
Table remaining capacity: 0
Waiting queue length: 0
